tests/test_cloud.py

```diff
@@ -1564,3 +1597,73 @@ def test_get_backup_succeeds_with_missing_main_file(self):
             backup_files[None].additional_files[0].path
             == "mt-backups/test-server/base/20210723T133818/data_0000.tar"
         )
+
+    @mock.patch("barman.cloud.CloudInterface")
+    def test_cloud_backup_catalog_has_keep_manager_capability(
+        self, cloud_interface_mock
+    ):
+        """
+        Verifies that KeepManagerMixinCloud methods are available in CloudBackupCatalog
+        and that they work as expected.
+
+        We deliberately do not test the functionality at a more granular level as
+        KeepManagerMixin has its own tests and CloudBackupCatalog adds no extra
+        functionality.
+        """
+        test_backup_id = "20210723T095432"
+
+        # Create a minimal in-memory CloudInterface implementation
```

Comment:
Maybe this minimal  implementation could be created in an external function to keep a better understanding of the test

Reply:
That's a good idea - I'll add a commit to do that